pallets/handles/src/lib.rs

```diff
@@ -1,31 +1,21 @@
-//! # Handles Pallet
-//! The Handles pallet provides functionality for claiming and retiring user handles.  Each MSA can have one user handle
-//! associated with it.  The handle consists of a canonical base, a "." delimiter and a unique numeric suffix.  It also has
-//! a display name.  e.g. user.734 with a display of "User"
+//! Unique human-readable strings for MSAs
 //!
+//! ## Quick Links
 //! - [Configuration: `Config`](Config)
 //! - [Extrinsics: `Call`](Call)
+//! - [Runtime API: `HandlesRuntimeApi`](../pallet_handles_runtime_api/trait.HandlesRuntimeApi.html)
+//! - [Custom RPC API: `HandlesApiServer`](../pallet_handles_rpc/trait.HandlesApiServer.html)
 //! - [Event Enum: `Event`](Event)
 //! - [Error Enum: `Error`](Error)
+#![doc = include_str!("../README.md")]
 //!
-//! ## Overview
+//! ## Shuffled Sequences
 //!
-//! ## Features
+//! To limit the value low or particular suffixes, this uses a suffled sequence to choose a suffix for a specific canonical handle string.
```

Comment:
"To limit the value low or particular suffixes"
not sure what this is supposed to be saying...
```suggestion
//! To limit the value low or particular suffixes, this uses a shuffled sequence to choose a suffix for a specific canonical handle string.
```

Reply:
Revised